media/js/firefox/sticky-promo.js

```diff
@@ -5,6 +5,13 @@
 (function() {
     'use strict';
 
+    // Check if promo exists on the page or if smaller than tablet.
```

Comment:
Seeing as this script is self-contained, it might make sense to treat it as a form of progressive enhancement. Suggest moving all of the logic here inside a `Mozilla.run()` callback?

https://bedrock.readthedocs.io/en/latest/browser-support.html#feature-detection-using-javascript